src/handlers/abstract/AbstractHandler.ts

```diff
@@ -62,11 +63,12 @@ export default abstract class AbstractHandler {
             } else {
                 // TODO: We shouldn't get here. This message should be handled by the Regression handler.
                 // Right now, we'll only get here for a Regression, but this should be fixed.
+                const reflection = getReflection()
                 const nfChat = getChannelFromClient(discordUser.client, config.channels.nf)
                 ;(nfChat as TextChannel).send(
                     `Attention! ${tagU(discordUser.user.id)} was demoted from ${prevRank.name} to ${
                         newRank.name
-                    } due to a relapse.`,
+                    } due to a relapse.\n${reflection}`,
```

Comment:
If you look at `src/data/reflections.ts`, you'll see that most of the strings are written as "Did you...". However, this message doesn't directly address the person who relapsed. Should we consider tagging the person before the reflection? That would look like:

`@PersonWhoRelapsed, did you have a recovery plan set in place?"

Reply:
Instead of at-ing them, what about their username? As it currently stands, this message will weird quite weirdly.

> Attention! @Foobar was demoted from A to B due to a relapse.
> 
> Did you have a recovery plan in place?

Would be better if it were like `Foobar, did you have a ....`?

Reply:
We will have to lowercase all the reflections in data/reflections. Other than that, I don't really see a lot of problems in implementing it this way.